src/plugins/intel_cpu/src/nodes/mkldnn_reduce_node.h

```diff
@@ -45,6 +45,7 @@ struct jit_reduce_post_call_args {
     size_t oc_off;          // offset in byte along channel on output tensor
     size_t channel_size;    // only for post ops fusion of nspc layout
     const float *divisor;   // mean = sum / divisor
+    const void* post_op_data;
```

Comment:
minor: ideologically this is `const void** post_op_data`

Reply:
Applied. Thanks Dmitry!